.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: k6 Community Forum
+    url: https://community.k6.io/
+    about: Please ask and answer questions here.
+  - name: k6 Documentation
+    url: https://github.com/grafana/k6-docs
```

Comment:
```suggestion
  - name: xk6-browser Documentation
    url: https://k6.io/docs/javascript-api/xk6-browser/
```

Reply:
This link is meant for adding documentation issues, so linking to the k6-docs repo is correct.